src/main.ts

```diff
@@ -10,7 +10,6 @@ import { plugin } from "./plugin";
  */
 export async function run() {
   const payload = github.context.payload.inputs;
-
   const env = Value.Decode(envSchema, payload.env);
```

Comment:
This will always crash on run:
https://github.com/Meniole/issue-comment-embeddings/actions/runs/10628609152/job/29463778679#step:5:45

The environment is populated either from GitHub or the Worker. Since this runs as an action, it should be loaded directly within `process.env`, which you can add inside `payload.env` if you want. In the case of a worker, it is send within the payload by Cloudflare.

Reply:
But, as mentioned before, I expect this to run in a Cloudflare worker environment. The env variables are sent within the payload.

Reply:
I think running as worker is fine.